screens/RiskFactors/RiskFactors.js

```diff
@@ -137,30 +141,43 @@ const styles = StyleSheet.create({
   },
   header: {
     color: 'white',
-    fontSize: 36,
+    fontSize: width * 0.09,
     fontWeight: 'bold',
-    marginBottom: height * 0.001,
     marginTop: height * 0.001
   },
+  instruction: {
+    color: 'white',
+    fontSize: width * 0.05,
+    textAlign: 'center'
+  },
+  questionContiner: {
+    alignContent: 'space-between',
+    alignItems: 'center',
+    flex: 1,
+    justifyContent: 'space-around',
+    width: '100%'
+  },
   questionText: {
-    alignSelf: 'flex-start',
-    fontSize: width * 0.030
+    fontSize: width * 0.030,
   },
   riskFactorCard: {
     alignItems: 'center',
+    backgroundColor: 'white',
     borderRadius: 15,
-    justifyContent: 'center',
     flex: 0.15,
-    marginBottom: height * 0.001,
-    marginTop: height * 0.001,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: 10,
```

Comment:
:grimacing:
Another instance of wanting to make this as dynamic/responsive as possible